podcast/2.12.m

```diff
@@ -0,0 +1,933 @@
+
+And so, both these things further cemented that this is going to be a wave bigger than the internet, bigger than the mobile phone. And I didn't really know what to do with that, to be honest. After Represently but before Roboflow, I toyed around with a bunch of different projects. I did some consulting work with Google, such as TensorFlow docs and freelance stuff. I worked on a text project for a financial company. I worked on a credit risk engine for a startup. And then one of the projects that I was attracted to, and you alluded to this earlier, was building something that would help value and sell farms, which I thought would probably be something that I might work on for a bit.
+
+But the broad unifying theme here is taking machine learning and applying it to a domain or a sector that hadn’t previously have it so that it unlocks unrealized potential for that sector, whether that's folks that want to have access to credit or people that want to be able to sort through mail more effectively, or valuing properties to make it be easier for real estate transactions and rural empowerment to exist.
+
+And I'll talk about how that turned into to Roboflow, but I know you want to talk about the farm thing too and I'm happy to share.
+
+**Beyang Liu:**
+
+Say more about the farm thing, because that was really interesting to me. And you come from a family, your family has a family farm in Iowa, right?
+
+**Joseph Nelson:**
+
+Yeah, Nelson Family Farms.
+
+**Beyang Liu:**
+
+Cool, so it runs in your blood. Was this project a thing to help your family manage their real estate holdings or was it something different?
+
+**Joseph Nelson:**
+
+I wish. That makes it sound so official. Yeah, like NFF in our real estate portfolio. Yeah, hardly. It was-
+
+**Beyang Liu:**
+
+It's actually a big multinational conglomerate–it just has a family fronting the thing.
+
+**Joseph Nelson:**
+
+Yeah, yeah.
+
+No, meanwhile, Doc Nelson, my dad, doesn't even know how to use Facebook. We're a pretty humble operation.
+
+But to be clear, I'm a fake farmer. I grew up in west Des Moines but my family would go to the farm with some frequency. But this came from one of my older brothers, who was at a farm auction one day. At this farm auction, they were auctioning off four 80 acre plots, which are called sections. And these four 80 acre sections were adjacent to one another, and the Southeastern most plot had a river running through it. And there is somewhat of a debate about how to value that plot because of how much of the land was tillable, arable land that you could use for productive agriculture purposes.
```

Comment:
```suggestion
But to be clear, I'm a fake farmer. I grew up in west Des Moines but my family would go to the farm with some frequency. But this came from one of my older brothers, who was at a farm auction one day. At this farm auction, they were auctioning off four 80 acre plots, which are called sections. And these four 80 acre sections were adjacent to one another, and the southeastern most plot had a river running through it. And there is somewhat of a debate about how to value that plot because of how much of the land was tillable, arable land that you could use for productive agriculture purposes.
```